src/module-sdk/1_installModule.ts

```diff
@@ -57,6 +57,9 @@ const installModule = async () => {
     chainId: baseSepolia.id,
   });
 
+  emailRecovery.address = config.addresses.universalEmailRecoveryModule;
```

Comment:
New module was deployed for sepolia?

Reply:
It was not, I did this when I was dealing with it on odyssey. Thought of keeping it in case we had to deploy the new module. 

Should we remove it ?

Reply:
Got it. Yeh maybe a comment explaining why it is needed.

When we run an e2e test, that would be on testnet and would require a new deployment